xamarin/Xamarin.Forms
# fix-9992 - Android - CollectionView - VerticalItemSpacing and HorizontalItemSpacing was not set for the new added item

### Description of Change ###
fix-9992 - Android - CollectionView - VerticalItemSpacing and HorizontalItemSpacing was not set for the new added item

### Issues Resolved ### 


- fixes #9992

### API Changes ###
 None

### Platforms Affected ### 
- Android

### Behavioral/Visual Changes ###
Android - CollectionView - VerticalItemSpacing and HorizontalItemSpacing was not set for the new added item

### Before/After Screenshots ### 

### Broken
![vertical_broken](https://user-images.githubusercontent.com/35541212/97110076-7adcc300-16d7-11eb-91b0-e33166ee8fd3.gif)

![horizontal_broken](https://user-images.githubusercontent.com/35541212/97110085-8203d100-16d7-11eb-9d86-476abee19057.gif)

### Fixed
![vertical_fixed](https://user-images.githubusercontent.com/35541212/97110104-a2339000-16d7-11eb-9bc9-979ffedd3f39.gif)

![horizontal_fixed](https://user-images.githubusercontent.com/35541212/97110252-6baa4500-16d8-11eb-90a1-938890fe7179.gif)


Not applicable

### Testing Procedure ###
Just add a new item to the `CollectionView` and check if the item spacing is applied

### PR Checklist ###
